Wikipedia/Code/WMFTableOfContentsPresentationController.swift

```diff
@@ -30,6 +30,11 @@ public class WMFTableOfContentsPresentationController: UIPresentationController
         return view
         }()
 
+    // MARK: - Status Bar Background
+
+    var statusBarBg: UIView = UIView()
+
```

Comment:
make this a `lazy var` like `backgroundView` instead of initializing here and then re-setting on L64. you can add it as a subview during `backgroundView`'s (lazy) initialization